internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -351,6 +351,28 @@ func NewPodForCR(cr *operatorv1beta1.Cryostat, specs *ServiceSpecs, imageTags *I
 		volumes = append(volumes, eventTemplateVolume)
 	}
 
+	// Add Auth properties as a volume if specified
+	if cr.Spec.AuthProperties != (operatorv1beta1.AuthPropertiesConfigMap{}) {
```

Comment:
Might as well make `cr.Spec.AuthProperties` a pointer and then you can just compare against `nil`.

Reply:
Oh yess that's better!